spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -232,6 +232,8 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private volatile int defaultTxSize = DEFAULT_TX_SIZE;
 
+	protected volatile boolean defaultDurableSubscription = true;
+
```

Comment:
Ok. I see what you are saying. The question for me is whether `durableSubscription` should be global or not. I believe that we should prevent proliferating global properties that apply only for certain types of middleware, or rather set up a contract for default behaviour (as in this case) and force binders to deal with it or provide options to override it if necessary. The tie-breaker, in my opinion, is whether the default setting makes sense from a logical point of view wrt the SCSt model (e.g. making groups durable by default for Rabbit ensures that it is consistent with Kafka, where a group subscription has the same effect). But I agree what we shouldn't ignore what we would expect to be global settings either (e.g. no binder can ignore the `groups` functionality for instance). My take was that if someone used a JMS binder, they would (and should) have the binder specific properties documented (which we don't, but should before 1.0.0.GA), so that they understand what are the options that they have for their own binder.